package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-api",
-  "version": "8.1.4",
+  "version": "9.0.0",
```

Comment:
Unlike our platform components, there's only one stream of development for this project, so it doesn't need a major bump on master for that reason. All of the dependency version bumps are minor or patch. Should this be a minor bump instead of a major one? Are there any breaking changes?

Since this is a module that others will use, I think it's more important that we stick as close as possible to semantic versioning than in some of our other projects.